packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.mjml

```diff
@@ -1,38 +1,64 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
+<%# This Source Code Form is subject to the terms of the Mozilla Public 
+  # License, v. 2.0. If a copy of the MPL was not distributed with this 
   # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
 <mj-include path="./css/appBadges/index.css" type="css" css-inline="inline" />
 
 <mj-section>
-  <mj-group>
+  <mj-group css-class="app-badges">
+    <% if (locals.iosUrl) { %>
     <mj-column>
-      <mj-image css-class="app-badges"
+      <mj-image
+        css-class="app-badge"
         href="<%- iosUrl %>"
         src="https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png"
-        alt="iOS badge">
+        alt="iOS badge"
+      >
       </mj-image>
     </mj-column>
+    <% } %> 
+    <% if (locals.androidUrl) { %>
     <mj-column>
-      <mj-image css-class="app-badges"
+      <mj-image
+        css-class="app-badge"
         href="<%- androidUrl %>"
         src="https://accounts-static.cdn.mozilla.net/product-icons/google-play.png"
-        alt="android badge">
+        alt="android badge"
+      >
       </mj-image>
     </mj-column>
+    <% } %>
   </mj-group>
 </mj-section>
 
+<% if (!locals.hideDeviceLink) { %>
 <mj-section>
```

Comment:
I think if you make the condition in the class similar to line 36, then you don't have to add `hideDeviceLink` in `fxa-auth-server/lib/senders/email.js`...